src/main/java/com/gooddata/md/report/MetricElement.java

```diff
@@ -25,7 +25,7 @@
     private final String drillAcrossStepAttributeDisplayFormUri;
 
     @JsonCreator
-    public MetricElement(@JsonProperty("uri") final String uri,
+    MetricElement(@JsonProperty("uri") final String uri,
```

Comment:
are you sure we want to hide the only possibility to create MetricElement with specified `format` and `drillAcrossStepAttributeDF`